### PR TITLE
pinniped.dev

### PR DESCRIPTION
**Pull request by aslafy-z**
package for https://github.com/vmware-tanzu/pinniped cli tool

### CODE REVIEW

File: projects/pinniped.dev/package.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+display-name: pinniped
+
+versions:
+  github: vmware-tanzu/pinniped/releases/tags
+
+platforms:
+  - linux/x86-64
+  - darwin/x86-64
+  - win32/x86-64
```
Comment (aslafy-z) [inline projects/pinniped.dev/package.yml:13]:
```suggestion
```

File: projects/pinniped.dev/package.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  script:
+   - go mod download
+   - mkdir -p "{{ prefix }}"/bin
+   - go build -trimpath -ldflags="$LDFLAGS" -o "{{ prefix }}"/bin/pinniped ./cmd/pinniped
+  env:
+    CGO_ENABLED: 0
+    LDFLAGS:
+      - -w
+      - -s
+      - -X go.pinniped.dev/internal/pversion.gitVersion=${{version}}
```
Comment (aslafy-z) [inline projects/pinniped.dev/package.yml:29]:
```suggestion
      - -X go.pinniped.dev/internal/pversion.gitVersion={{version}}
```

File: projects/pinniped.dev/package.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+   - go build -trimpath -ldflags="$LDFLAGS" -o "{{ prefix }}"/bin/pinniped ./cmd/pinniped
+  env:
+    CGO_ENABLED: 0
+    LDFLAGS:
+      - -w
+      - -s
+      - -X go.pinniped.dev/internal/pversion.gitVersion={{version}}
+
+test:
+  - pinniped | grep "Pinniped CLI"
```
Comment (aslafy-z) [inline projects/pinniped.dev/package.yml:32]:
```suggestion
  - pinniped | grep "Pinniped CLI is the client-side binary"
```

File: projects/pinniped.dev/package.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  env:
+    CGO_ENABLED: 0
+    LDFLAGS:
+      - -w
+      - -s
+      - -X go.pinniped.dev/internal/pversion.gitVersion={{version}}
+
+test:
+  - pinniped | grep "Pinniped CLI is the client-side binary"
+  - echo $(pinniped version || true) | grep "v{{version}}"
```
Comment (aslafy-z) [inline projects/pinniped.dev/package.yml:33]:
```suggestion
  - echo $(pinniped version || true) | grep "{{version}}"
```

**Comment by aslafy-z** [comment]
~aarch64 is not officially supported~, that's why I did not added it in the first time... What is the pkgx plan for that?
Can you please add comments to the PR instead or editing it? I had pending changes and it's a bit confusing. :pray: 

edit: looks like the changes were done: https://github.com/vmware-tanzu/pinniped/issues/1386#issuecomment-1749764274. Pinniped will start supporting these!

**Comment by aslafy-z** [comment]
Looks like the next release will fully support it for the CLI as per https://github.com/vmware-tanzu/pinniped/issues/1386#issuecomment-1749764274. Let's make this PR wait a few days and see!

**Comment by jhheider** [comment]
> Can you please add comments to the PR instead or editing it?

Sure. Apologies. In general, if it builds and runs, we package it.

**Pull request merged**